### PR TITLE
Fix markdown for .env section

### PR DESCRIPTION
**Pull request by Elitesparkle**
Due to consecutive markdown blocks not being rendered correctly inside lists (only when accessing the page from the Pycord Guide), I removed one of the two examples that are basically showing the same thing in the same place. In addition to that, I improved the explanation so that it is more clear for beginners.

### CODE REVIEW

File: docs/getting-started/creating-your-first-bot.mdx [hunk 1]
```diff
@@ -90,28 +90,31 @@ getting leaked is to store it in `.env` files.
...
-3.  Install [`python-dotenv`](https://pypi.org/project/python-dotenv/)
+3.  Install [`python-dotenv`](https://pypi.org/project/python-dotenv/).
     ```bash
     python -m pip install python-dotenv
     ```
+4. Load the token from the `.env` file.
+   ```python
+   dotenv.load_dotenv()
+   token = str(os.getenv("TOKEN"))
+5. Pass your token as parameter when rurnning the bot.
```
Comment (BobDotCom) [inline docs/getting-started/creating-your-first-bot.mdx:106]:
```suggestion
5. Pass your token as parameter when running the bot.
```

File: docs/getting-started/creating-your-first-bot.mdx [hunk 2]
```diff
@@ -90,28 +90,31 @@ getting leaked is to store it in `.env` files.
...
     TOKEN = NzkyNzE1NDU0MTk2MDg4ODQy.X-hvzA.Ovy4MCQywSkoMRRclStW4xAYK7I
     ```
-3.  Install [`python-dotenv`](https://pypi.org/project/python-dotenv/)
+3.  Install [`python-dotenv`](https://pypi.org/project/python-dotenv/).
     ```bash
     python -m pip install python-dotenv
     ```
+4. Load the token from the `.env` file.
+   ```python
+   dotenv.load_dotenv()
```
Comment (BobDotCom) [inline docs/getting-started/creating-your-first-bot.mdx:104]:
```suggestion
   import dotenv
   dotenv.load_dotenv()
```

**Review by BobDotCom — CHANGES_REQUESTED** [review]

**Comment by BobDotCom** [comment]
Please fix merge conflicts

**Review by Elitesparkle — COMMENTED** [review]
Add missing import and fix typo

**Review by Lulalaby — APPROVED** [review]

**Pull request merged**